packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -410,24 +422,100 @@ contract Deploy is Deployer {
         addr_ = safe;
     }
 
+    /// @notice Deploy a LivenessModule and LivenessGuard for use on the Security Council Safe
+    function _deployLivenessModuleAndGuardian() internal returns (address module_, address guard_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        guard_ = address(new LivenessGuard(councilSafe));
+        console.log("New LivenessGuard deployed at %s", address(guard_));
+
+        address fallbackOwner = mustGetAddress("SystemOwnerSafe");
+        module_ = address(
+            // todo: set these values in the deploy config???
+            new LivenessModule({
+                _safe: councilSafe,
+                _livenessGuard: LivenessGuard(guard_),
+                _livenessInterval: 1,
+                _thresholdPercentage: 20,
+                _minOwners: 1,
+                _fallbackOwner: fallbackOwner
+            })
+        );
+        console.log("New LivenessModule deployed at %s", address(module_));
+    }
+
+    /// @notice Deploy a Security Council with LivenessModule and LivenessGuard.
+    function deploySecurityCouncilSafe() public broadcast returns (address addr_) {
+        console.log("Deploying Security Council Safe");
+        (SafeProxyFactory safeProxyFactory, Safe safeSingleton) = _getSafeFactory();
+
+        address[] memory initialSigners = new address[](1);
+        initialSigners[0] = msg.sender;
+
+        bytes memory initData = abi.encodeCall(
+            Safe.setup, (initialSigners, 1, address(0), hex"", address(0), address(0), 0, payable(address(0)))
+        );
+        Safe safe = Safe(
+            payable(
+                address(safeProxyFactory.createProxyWithNonce(address(safeSingleton), initData, block.timestamp + 1))
+            )
+        );
+
+        save("SecurityCouncilSafe", address(safe));
+        console.log("New SecurityCouncilSafe deployed at %s", address(safe));
+        (address module, address guard) = _deployLivenessModuleAndGuardian();
+
+        _callViaSafe({ _safe: safe, _target: address(safe), _data: abi.encodeCall(GuardManager.setGuard, (guard)) });
+        console.log("LivenessGuard setup on SecurityCouncilSafe");
+        _callViaSafe({ _safe: safe, _target: address(safe), _data: abi.encodeCall(ModuleManager.enableModule, (module)) });
+        console.log("LivenessModule enabled on SecurityCouncilSafe");
+
+        // Add a more realistic number of signers to the Council Safe
+        uint256 numSigners = 2 * LivenessModule(module).minOwners();
+
+        // There is already one signer, so we need to add numSigners - 1 additional signers.
+        // During a production deployment, the signer used during setup should be removed at the end, however
+        // this is not feasible here because it would require us to reach a threshold of signatures.
+        address[] memory additionalSigners = new address[](numSigners - 1);
+
+        // Add the additional signers
```

Comment:
I am confused why we are adding random signers here? I thought this script would be able to be use in production

Reply:
My intention was for it to be useful as an example. If we want to be able to specify the Signer set, how should we go about that? 

Options: 
- Add to the deploy config
- default to none and provide as env var
- add another config file

Reply:
Actually the same issue also applies to the other params of the [LivenessModule from above](https://github.com/ethereum-optimism/optimism/pull/10120/files#r1566374763).

Reply:
I see 2 options, 1) add to deploy config or 2) break out into its own script and create a new way to configure that new script. There is a question around when we set up a new testnet, what will the friction be for  configuring it to be exactly like mainnet

Reply:
How were you planning on deploying this to production if not use the deploy script here?

Reply:
Good question. My intent was to provide the scripts here as an example, but as you can see from what I've done the existing constructor checks make it impossible to configure the liveness module without multiple signatures.

I think this is fine though, as I'm hesitant to remove constructor checks at this point, and I don't think we need a scalable solution to Security Council deployment.